receiver/aerospikereceiver/metadata.yaml

```diff
@@ -94,6 +119,15 @@ metrics:
       input_type: string
       monotonic: false
       aggregation: cumulative
+  aerospike.node.query.tracked:
```

Comment:
I have it separate because it is a node level metric, the other query metrics are namespace level. I think that makes sense, let me know if you disagree.

Reply:
Makes perfect sense 👍